netmaster/objApi/apiController.go

```diff
@@ -79,6 +80,16 @@ func NewAPIController(router *mux.Router) *APIController {
 	return ctrler
 }
 
+// Utility function to check if string exists in a slice
+func stringInSlice(a string, list []string) bool {
```

Comment:
This was required for searching list of policy names in EndpointGroup object. I couldn't build the map since the object comes from contivModel(or i had to build the map each time i need to search it), We don't expect hundreds of policy names per endpoint group. so, performance here should be OK.